spec/models/user_spec.rb

```diff
@@ -6,7 +6,7 @@
     expect { User.make!(:username => "") }.to raise_error
     expect { User.make!(:username => "*") }.to raise_error
 
-    User.make!(:username => "test")
+    User.make!(:username => "test") unless User.find_by(username: 'test')
```

Comment:
Why is this needed? Is this papering over a test failing to clean the database?

Reply:
This fixed a failure on two runs:
* https://travis-ci.org/lobsters/lobsters/builds/283332809?utm_source=github_status&utm_medium=notification
* https://travis-ci.org/lobsters/lobsters/builds/283330930?utm_source=github_status&utm_medium=notification

We have transnational fixtures turned on, so we should be clearing the database after each run.

Reply:
Uh... that sure looks like either test data failed to get cleaned or some general pre-test hook is creating duplicate data. There's something wrong here, this `unless` shouldn't exist. I'm sorry I didn't think of this before hitting merge.

Reply:
Ill dig at this more locally and try to see whats causing the issue.